### PR TITLE
Fix #4: Adds Issue Template

### LINKED ISSUE #4: Issue Template
Add Issue Template

References:
https://github.com/oppia/oppia-android/tree/develop/.github/ISSUE_TEMPLATE
https://github.com/forem/forem/tree/main/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by rray524**
Fix #4  : Adds Issue Template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG]: "
+labels: [triage needed, bug]
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:4]: Remove labels from both the templates.

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    id: screenshots-or-videos
+    attributes:
+      label: Screenshots/Videos
+      description: |
+        If applicable, add screenshots or videos to help explain your problem.
+
+        **Tip:** You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false
+  - type: input
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:45]: Update this entire section to get browser and device information like 
Macbook Pro M1
Chrome xx.yy.zz version
etc.

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    id: device
+    attributes:
+      label: What device/emulator are you using?
+      description: Please specify the device/emulator you're using.
+      placeholder: OnePlus Nord2 5G
+    validations:
+      required: false
+  - type: input
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:53]: This is not required. Please don't directly copy paste try to understand and apply.

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    id: android-version
+    attributes:
+      label: Which Android version is your device/emulator running?
+      description: Please specify the Android version that is running on your device/emulator.
+      placeholder: Android 11
+    validations:
+      required: false
+  - type: input
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:61]: This is not required.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Update title and description to:

Fix #4: Adds Issue Template

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 1 above)
Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml]: removed. thanks

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 2 above)
Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml:44]: resolved. thanks.

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 3 above)
Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml:52]: resolved. thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
There are still some issues as I am able to see errors on this link:

https://github.com/rt4914/KlinicCon-Frontend/blob/c3f3bc52110db84697f5b57360f81ab5f530aa6d/.github/ISSUE_TEMPLATE/1_bug_report_form.yml

**Comment by rray524** [comment]
> There are still some issues as I am able to see errors on this link:
> 
> https://github.com/rt4914/KlinicCon-Frontend/blob/c3f3bc52110db84697f5b57360f81ab5f530aa6d/.github/ISSUE_TEMPLATE/1_bug_report_form.yml

error fixed. I checked commit preview and it works fine. Thank you!

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
+name: Fix #4: Adds Issue Template
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:1]: Update this to `Bug Report`

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+description: Fix #4: Adds Issue Template
```
Comment (rt4914) [inline .github/ISSUE_TEMPLATE/1_bug_report_form.yml:2]: description: Mention details of what this bug is

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks @rray524 Looks in really good shape.

Couple of things 
1. Do not resolve the conversation on your own. The person who has started the comment-thread will resolve it. 
2. I think you missed this from my previous review https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2223034274

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 4 above)
Comment (rohit-simbanic) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml:60]: I removed this one too. Thanks @rt4914

Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml:60]: I removed it too. thanks @rt4914

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 5 above)
Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml]: updated. thanks

File: .github/ISSUE_TEMPLATE/1_bug_report_form.yml (hunk 6 above)
Comment (rray524) [reply .github/ISSUE_TEMPLATE/1_bug_report_form.yml]: updated this too. thanks @rt4914

**Review by rt4914 — CHANGES_REQUESTED** [review]
@rray524 Thanks. The code is perfect but I cannot merge it until you fix this one thing. I have mentioned this couple of times in my previous reviews:

First Mention: https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2223034274
Second Mention: https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2227083979

Basically you need to update the title to `Fix #4: Adds Issue Template`

**Comment by rray524** [comment]
> @rray524 Thanks. The code is perfect but I cannot merge it until you fix this one thing. I have mentioned this couple of times in my previous reviews:
> 
> First Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2223034274) Second Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2227083979)
> 
> Basically you need to update the title to `Fix #4: Adds Issue Template`

sorry for the misunderstanding, I thought title is similar to name. 
Now `title: "Fix #4: Adds Issue Template"` added this as per requirements.

**Comment by rt4914** [comment]
> > @rray524 Thanks. The code is perfect but I cannot merge it until you fix this one thing. I have mentioned this couple of times in my previous reviews:
> > First Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2223034274) Second Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2227083979)
> > Basically you need to update the title to `Fix #4: Adds Issue Template`
> 
> sorry for the misunderstanding, I thought title is similar to name. Now `title: "Fix #4: Adds Issue Template"` added this as per requirements.

@rray524 Title can be different from issue-title but even if it remains same then also the issue title is `Issue Template` so from that perspective also it is incorrect. That being said the title and description of this PR are still not updated. This is how it looks for me.
Screenshot 2024-08-10 at 8 18 06 PM

**Comment by rray524** [comment]
> > > @rray524 Thanks. The code is perfect but I cannot merge it until you fix this one thing. I have mentioned this couple of times in my previous reviews:
> > > First Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2223034274) Second Mention: [#12 (review)](https://github.com/rt4914/KlinicCon-Frontend/pull/12#pullrequestreview-2227083979)
> > > Basically you need to update the title to `Fix #4: Adds Issue Template`
> > 
> > 
> > sorry for the misunderstanding, I thought title is similar to name. Now `title: "Fix #4: Adds Issue Template"` added this as per requirements.
> 
> @rray524 Title can be different from issue-title but even if it remains same then also the issue title is `Issue Template` so from that perspective also it is incorrect. That being said the title and description of this PR are still not updated. This is how it looks for me. Screenshot 2024-08-10 at 8 18 06 PM

This is after changing the title and description, will it work @rt4914 ?
Thanks for your suggestions again.

image

**Review by rt4914 — APPROVED** [review]
Congratulations on your First PR. 🚀

**Pull request merged**